src/main/java/org/terracotta/statistics/AbstractOperationStatistic.java

```diff
@@ -36,6 +36,8 @@
   @ContextAttribute("properties") public final Map<String, Object> properties;
   @ContextAttribute("type") public final Class<T> type;
 
+  private final ThreadLocal<Long> operationStartTime = new ThreadLocal<>();
```

Comment:
We should probably document that an operation needs to stop and start in the same thread.

Reply:
yes this was not documented, I agree.